packages/phishing-controller/src/utils.test.ts

```diff
@@ -603,3 +604,47 @@ describe('getHostname', () => {
     expect(getHostnameFromUrl(url)).toBe(expectedHostname);
   });
 });
+
+describe('extractDomainName', () => {
+  it('should extract the primary domain from a standard hostname', () => {
+    const hostname = 'www.example.com';
+    const expected = 'example.com';
+    const result = extractDomainName(hostname);
+    expect(result).toBe(expected);
+  });
+
+  it('should extract the primary domain from a hostname with multiple subdomains', () => {
```

Comment:
Should we add test cases for things like site.web.google.com? essentially testing for longer hostnames